test/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ macro (add_app_test test_filename test_alternative)
     # Add the test to its general target (cli or api).
     if (${test_alternative} STREQUAL "CLI_TEST")
         add_dependencies (${target} "${PROJECT_NAME}") # cli test needs the application executable
+        target_include_directories(${target} PUBLIC "${SEQAN3_INCLUDE_DIR}/../test/include")
```

Comment:
Just a question: Why don't we have the source directory as a variable? Would make the path a bit more readable.

Reply:
Thank you for this suggestion; I found a solution with SEQAN3_CLONE_DIR which makes it indeed better readable.